fs/fs.go

```diff
@@ -169,17 +216,39 @@ func processMounts(mounts []*mount.Info, excludedMountpointPrefixes []string) ma
 			}
 		}
 
+		uuid := ""
+		if u, found := deviceNameToFsUUID[mount.Source]; found {
+			uuid = u
+		}
+
 		partitions[mount.Source] = partition{
 			fsType:     mount.Fstype,
 			mountpoint: mount.Mountpoint,
 			major:      uint(mount.Major),
 			minor:      uint(mount.Minor),
+			uuid:       uuid,
 		}
 	}
 
 	return partitions
 }
 
+func (self *RealFsInfo) getFsUUIDFromDeviceName(deviceName string) string {
+	if uuid, found := self.deviceNameToFsUUID[deviceName]; found {
+		return uuid
+	}
+	return ""
+
+}
+
+func (self *RealFsInfo) getDeviceNameFromFsUUID(uuid string) string {
```

Comment:
I dont think this is neccessary.  If we are just going to check to see if device == "" in GetDeviceInfoByFsUUID() anyways, we might as well just do self.fsUUIDToDeviceName[uuid] there.